netbout-web/src/main/java/com/netbout/rest/MarkdownTxtmark.java

```diff
@@ -72,4 +80,25 @@ private static String formatLinks(final String txt) {
         matcher.appendTail(result);
         return result.toString();
     }
+    /**
+     * Insert two spaces before a new line symbol to force html line break.
+     * @param txt Text to replace
+     * @return Text with Markdown-formatted links
+     */
+    private static String makeLineBreak(final String txt) {
+        final StringBuffer result = new StringBuffer();
+        final Matcher matcher = MarkdownTxtmark.NEW_LINE.matcher(txt);
+        while (matcher.find()) {
+            if (!matcher.hitEnd()) {
+                matcher.appendReplacement(
+                    result,
+                    String.format(
+                        "%s  ", matcher.group()
```

Comment:
@dmzaytsev Javadoc says about inserting two spaces before new line symbol, but here I see two spaces inserted AFTER match. Can you explain?